hera_qm/tests/test_uvflag.py

```diff
@@ -602,6 +602,37 @@ def test_to_waterfall_bl_multi_pol():
     nt.assert_true(uvf2.polarization_array[0] == ','.join(map(str, uvf.polarization_array)))
 
 
+def test_collapse_pol():
+    uvf = UVFlag(test_f_file)
+    uvf.weights_array = np.ones_like(uvf.weights_array)
+    uvf2 = uvf.copy()
+    uvf2.polarization_array[0] = -4
+    uvf.__add__(uvf2, inplace=True, axis='pol')  # Concatenate to form multi-pol object
+    uvf2 = uvf.copy()
+    print(uvf2.polarization_array)
+    uvf2.collapse_pol()
+    nt.assert_true(len(uvf2.polarization_array) == 1)
+    print(uvf2.polarization_array)
+    print(uvf.polarization_array)
```

Comment:
can these print statements be removed? Were they just debugg stuff?

Reply:
whoops